### PR TITLE
Add locale in Dockerfile.base

### PR DESCRIPTION
**Pull request by nstdio**

### CODE REVIEW

File: Dockerfile.base [hunk 1]
```diff
@@ -50,3 +53,4 @@ RUN apt-get update && \
   && chmod +x /usr/local/bin/docker-compose \
   && rm -rf /var/lib/apt/lists/* \
   && rm -rf /tmp/*
+  && locale-gen ${LANG}
```
Comment (myoung34) [inline Dockerfile.base:56]: I think you also need to add `dpkg-reconfigure locales`

**Review by myoung34 — APPROVED** [review]

**Review by myoung34 — CHANGES_REQUESTED** [review]
I think you also need to add `dpkg-reconfigure locales`

**Comment by nstdio** [comment]
@myoung34 Can you please recheck?

File: Dockerfile.base [hunk 2]
```diff
@@ -31,6 +31,10 @@ RUN apt-get update && \
     wget \
     dirmngr \
     openssh-client \
+    locales \
+  && dpkg-reconfigure locales \
```
Comment (myoung34) [inline Dockerfile.base]: move this to after `locale-gen`

File: Dockerfile.base [hunk 3]
```diff
@@ -31,6 +31,10 @@ RUN apt-get update && \
     wget \
     dirmngr \
     openssh-client \
+    locales \
+  && dpkg-reconfigure locales \ 
+  && locale-gen en_US.UTF-8 \
+  && export LANG=en_US.UTF-8 LANGUAGE=en_US:en LC_ALL=en_US.UTF-8 \
```
Comment (myoung34) [inline Dockerfile.base]: these are lost on the running container, but i dont think theyre needed with the other 2 lines

**Review by myoung34 — CHANGES_REQUESTED** [review]

File: Dockerfile.base (hunk 3 above)
Comment (nstdio) [reply Dockerfile.base]: Yeah. Just double checking we remove this line and use `-e LC_ALL=en_US.UTF-8` when running the container?

Comment (myoung34) [reply Dockerfile.base]: I dont think so, but I'm unsure

**Review by myoung34 — CHANGES_REQUESTED** [review]
if you add 
```
ENV LANG=en_US.UTF-8                                                                                                                                         
ENV LANGUAGE=en_US:en                                                                                                                                        
ENV LC_ALL=en_US.UTF-8 
```

to `Dockerfile.base` (after line 8: `ENV DOCKER_COMPOSE_VERSION="1.27.4"`) I can merge this, I tested it and it worked as expected. It will also allow overrides with `-e` which is ideal for future locales

**Review by myoung34 — APPROVED** [review]

**Comment by myoung34** [comment]
Thanks!

**Pull request merged**